<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(19, 41, 61);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="gp_Tudo">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="7">
        <widget class="QFrame" name="Email">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="txt_Email">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imp_Email">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="7">
        <widget class="QFrame" name="Telefone">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="txt_telefone">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>76</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Telefone:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imp_telefone">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="inputMask">
             <string>(00)00000-0000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="7" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
        <widget class="QFrame" name="Cadastro">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="txt_Cadastro">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Cadastro</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QFrame" name="Nova_Senha">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="txt_Nova_Senha">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Nova Senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Input_Nova_Senha">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QFrame" name="Foto">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Botao_Add_Foto">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#Botao_Add_Foto {
    background-color: #0b3c58; /* Cor normal */
    color: #e8f1f2;
    border-radius: 5px;
    padding: 5px;
    border: none; /* Remove a borda padrão, se houver */
    cursor: pointer; /* Muda o cursor para indicar que é clicável */
    transition: background-color 0.3s, transform 0.1s; /* Efeito suave */
}

#Botao_Add_Foto:hover {
    background-color: #1B98E0; /* Cor quando o mouse estiver sobre o botão */
}

#Botao_Add_Foto:pressed {
    background-color: #0a2a3f; /* Cor quando o botão for pressionado */
    transform: scale(0.95); /* Efeito de leve redução para parecer um clique */
}
</string>
            </property>
            <property name="text">
             <string>Adicionar foto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pos_img">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="7">
        <widget class="QFrame" name="Confirmar_Senha">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="txt_Confirmar_Senha">
            <property name="minimumSize">
             <size>
              <width>135</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Confirmar Senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Input_Confirmar_Senha">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QFrame" name="Data_Nascimento">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="txt_Data">
            <property name="minimumSize">
             <size>
              <width>172</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>172</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Data de nascimento:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="date_data">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QFrame" name="Endereco">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="txt_Endereco">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Endereço:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imp_endereco">
            <property name="minimumSize">
             <size>
              <width>325</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="8">
        <widget class="QFrame" name="CPF">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="txt_CPF">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>CPF:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imp_CPF">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="inputMask">
             <string>000.000.000-00</string>
            </property>
            <property name="text">
             <string>..-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QFrame" name="Nome">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="txt_nome">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(232, 241, 242);</string>
            </property>
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imp_nome">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="10" column="7" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
        <widget class="QFrame" name="Entrar">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="Botao_entrar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#Botao_entrar {
    background-color: #0b3c58; /* Cor normal */
    color: #e8f1f2;
    border-radius: 5px;
    padding: 5px;
}

#Botao_entrar:hover {
    background-color: #1B98E0; /* Cor quando o mouse estiver sobre o botão */
}

#Botao_entrar:pressed {
    background-color: #0a2a3f; /* Cor quando o botão for pressionado */
    transform: scale(0.95); /* Efeito de leve redução para parecer um clique */
}
</string>
            </property>
            <property name="text">
             <string>Entrar</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="8">
        <widget class="QFrame" name="Sexo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(232, 241, 242);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="txt_Sexo">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sexo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <widget class="QRadioButton" name="opc_Femi">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Feminino</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="opc_Masc">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Masculino</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
            <widget class="QRadioButton" name="opc_Outro">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Outro</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagemTeste1.qrc"/>
 </resources>
 <connections/>
</ui>
